Nodejs/Product/Nodejs/PkgCmdId.cs

```diff
@@ -24,8 +24,12 @@ internal static class PkgCmdId
         public const int cmdidNpmOpenModuleHomepage = 0x306;
         public const int menuIdNpm = 0x3000;
 
-        public const int cmdidWorkSpaceNpmInstallMissing = 0x4001;
-        public const int cmdidWorkSpaceNpmInstallNew = 0x4002;
-        public const int cmdidWorkSpaceNpmUpdate = 0x4003;
+        public const int cmdidWorkSpaceNpmInstallMissing = 0x0200;
+        public const int cmdidWorkSpaceNpmInstallNew = 0x0201;
+        public const int cmdidWorkSpaceNpmUpdate = 0x0202;
+        public const int cmdidWorkSpaceNpmDynamicScript = 0x1000;
+
+        // only allow 100 scripts
+        public const int cmdidWorkSpaceNpmDynamicScriptMax = cmdidWorkSpaceNpmDynamicScript + 100;
```

Comment:
>100 [](start = 94, length = 3)

It might be nice to synthesize a bunch of dummy scripts and confirm that something sane happens just under/over the limit.